From 7e0068650712539e444b77042df56177b8fa6cb2 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Wed, 11 Aug 2021 23:26:30 +0300
Subject: [PATCH] [SW-0] gaudi: security: fix wrong tpc protection

TPC single page protection contained wrong increment,
which has caused undesired protection of 16 QM_SRF at
offset 0xE00 to 0xE40, which should not be protected.
This patch fix this wrong increment.

Change-Id: I4ec5922c68d9d2913a51c85753c464e06246c09e
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/gaudi/security.c b/gaudi/security.c
index 1abe9c6..fb9e025 100644
--- a/gaudi/security.c
+++ b/gaudi/security.c
@@ -1083,7 +1083,7 @@
 	/* offsets 0xA00 - 0xE80: unprotected */
 
 	/* offsets 0xE80 - 0xF80 */
-	prot_base += SNGL_REG_PROT_SIZE * 9;
+	prot_base += SNGL_REG_PROT_SIZE * 10;
 	set_prot_bits_range_all(prot_base, PROT_SIZE_MAX - (prot_base - base));
 }
 
